app/assets/javascripts/govuk_publishing_components/components/checkboxes.js

```diff
@@ -25,26 +25,27 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
       $(scope).on('change', 'input[type=checkbox]', function(e) {
         if (GOVUK.analytics && GOVUK.analytics.trackEvent) {
-          var $checkbox = $(e.target);
-          var category = $checkbox.data("track-category");
-          if (typeof category !== "undefined") {
-            var isChecked = $checkbox.is(":checked");
-            var uncheckTrackCategory = $checkbox.data("uncheck-track-category");
-            if (!isChecked && typeof uncheckTrackCategory !== "undefined") {
-              category = uncheckTrackCategory;
+          if(typeof e.suppressAnalytics === 'undefined' || e.suppressAnalytics !== true ) {
```

Comment:
Maybe add a short comment above this line to explain... `// where checkboxes are manipulated externally in finders, suppressAnalytics is passed to prevent duplicate GA events` ?